chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -277,14 +277,16 @@
   owner "crowbar"
   group "crowbar"
   mode "0644"
-  variables(web_host: "0.0.0.0",
-            web_port: node["crowbar"]["web_port"] || 3000,
-            user: "crowbar",
-            concurrency_model: "EventMachine",
-            group: "crowbar",
-            logdir: logdir,
-            logname: "production",
-            app_location: "/opt/dell/crowbar_framework")
+  variables(
+    web_host: "127.0.0.1",
+    web_port: node["crowbar"]["web_port"] || 3000,
+    user: "crowbar",
+    concurrency_model: "EventMachine",
+    group: "crowbar",
+    logdir: logdir,
+    logname: "production",
+    app_location: "/opt/dell/crowbar_framework"
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.